recipes/pyblake2/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = '1.1.0' %}
+{% set sha256 = '3a850036bf42053c74bfc52c063323ca78e40ba1f326b01777da5750a143631a' %}
+{% set name = 'pyblake2' %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+
+build:
+  number: 0
+  script:
+    - python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - toolchain
+    - msinttypes  # [win and py<35]
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - pyblake2
+  commands:
+    - conda inspect linkages {{ name|lower }}  # [unix]
+    - conda inspect objects {{ name|lower }}   # [osx]
+
+
+about:
+  home: https://github.com/dchest/pyblake2
+  license_family: MIT
+  license: MIT
+  license_file: COPYING
+  summary: |
+    pyblake2 is an extension module for Python implementing BLAKE2 hash function.
+    BLAKE2 is a cryptographic hash function, which offers highest security while being as fast as MD5 or SHA-1,
```

Comment:
Probably should end line with a `.` instead of `,`. That said, this is a really minor thing and not blocking.